src/blueapi/config.py

```diff
@@ -41,6 +41,15 @@ class StompConfig(BaseModel):
     auth: BasicAuthentication | None = None
 
 
+class TiledConfig(BaseModel):
+    """
+    Config for connecting to a tiled instance
+    """
+
+    uri: str
```

Comment:
Could: Can we use separate `host` and `port` fields to be consistent with the other config models?